package.json

```diff
@@ -232,5 +232,8 @@
     "react-tooltip": "^4.2.13",
     "recompose": "^0.30.0",
     "webpack-cli": "^4.7.2"
+  },
+  "browser": {
```

Comment:
Note, I was getting a webpack compile error without adding this. Here's that error for reference:

```
5:35:18 PM webpack.1 |  BREAKING CHANGE: webpack < 5 used to
5:35:18 PM webpack.1 |  >  include polyfills for node.js core modules by default.
5:35:18 PM webpack.1 |  This is no longer the case. Verify if you need this module a
5:35:18 PM webpack.1 |  >  nd configure a polyfill for it.
5:35:18 PM webpack.1 |  If you want to include a polyfill, you need to:
5:35:18 PM webpack.1 |  	- add a fallback 'resolve.fallback: { "crypto": require.res
5:35:18 PM webpack.1 |  >  olve("crypto-browserify") }'
5:35:18 PM webpack.1 |  	- install 'crypto-browserify'
5:35:18 PM webpack.1 |  If you don't want to include a polyfill, you can use an empt
5:35:18 PM webpack.1 |  >  y module like this:
5:35:18 PM webpack.1 |  	resolve.fallback: { "crypto": false }
5:35:18 PM webpack.1 |  resolve 'crypto' in '/Users/cody/dev/Spoke/src/extensions/se
5:35:18 PM webpack.1 |  >  cret-manager'
5:35:18 PM webpack.1 |    Parsed request is a module
5:35:18 PM webpack.1 |    using description file: /Users/cody/dev/Spoke/package.json
5:35:18 PM webpack.1 |  >   (relative path: ./src/extensions/secret-manager)
5:35:18 PM webpack.1 |      Field 'browser' doesn't contain a valid alias configurat
5:35:18 PM webpack.1 |  >  ion
5:35:18 PM webpack.1 |      resolve as module
5:35:18 PM webpack.1 |        /Users/cody/dev/Spoke/src/extensions/secret-m
5:35:18 PM webpack.1 |  >  anager/node_modules doesn't exist or is not a dire
5:35:18 PM webpack.1 |  >  ctory
5:35:18 PM webpack.1 |        /Users/cody/dev/Spoke/src/extensions/node_mod
5:35:18 PM webpack.1 |  >  ules doesn't exist or is not a directory
5:35:18 PM webpack.1 |        /Users/cody/dev/Spoke/src/node_modules doesn'
5:35:18 PM webpack.1 |  >  t exist or is not a directory
5:35:18 PM webpack.1 |        looking for modules in /Users/cody/dev/Spoke/node_modu
5:35:18 PM webpack.1 |  >  les
5:35:18 PM webpack.1 |          single file module
5:35:18 PM webpack.1 |            using description file: /Users/cody/dev/Spoke/pack
5:35:18 PM webpack.1 |  >  age.json (relative path: ./node_modules/crypto)
5:35:18 PM webpack.1 |              no extension
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/Spoke/node_modules/cr
5:35:18 PM webpack.1 |  >  ypto doesn't exist
5:35:18 PM webpack.1 |              .js
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/Spoke/node_modules/cr
5:35:18 PM webpack.1 |  >  ypto.js doesn't exist
5:35:18 PM webpack.1 |              .jsx
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/Spoke/node_modules/cr
5:35:18 PM webpack.1 |  >  ypto.jsx doesn't exist
5:35:18 PM webpack.1 |              .json
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/Spoke/node_modules/cr
5:35:18 PM webpack.1 |  >  ypto.json doesn't exist
5:35:18 PM webpack.1 |          /Users/cody/dev/Spoke/node_modules/crypto d
5:35:18 PM webpack.1 |  >  oesn't exist
5:35:18 PM webpack.1 |        looking for modules in /Users/cody/dev/node_modules
5:35:18 PM webpack.1 |          single file module
5:35:18 PM webpack.1 |            using description file: /Users/cody/dev/package.js
5:35:18 PM webpack.1 |  >  on (relative path: ./node_modules/crypto)
5:35:18 PM webpack.1 |              no extension
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/node_modules/crypto d
5:35:18 PM webpack.1 |  >  oesn't exist
5:35:18 PM webpack.1 |              .js
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/node_modules/crypto.j
5:35:18 PM webpack.1 |  >  s doesn't exist
5:35:18 PM webpack.1 |              .jsx
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/node_modules/crypto.j
5:35:18 PM webpack.1 |  >  sx doesn't exist
5:35:18 PM webpack.1 |              .json
5:35:18 PM webpack.1 |                Field 'browser' doesn't contain a valid alias
5:35:18 PM webpack.1 |  >  configuration
5:35:18 PM webpack.1 |                /Users/cody/dev/node_modules/crypto.j
5:35:18 PM webpack.1 |  >  son doesn't exist
5:35:18 PM webpack.1 |          /Users/cody/dev/node_modules/crypto doesn't
5:35:18 PM webpack.1 |  >   exist
5:35:18 PM webpack.1 |        /Users/cody/node_modules doesn't exist
5:35:18 PM webpack.1 |  >  22m or is not a directory
5:35:18 PM webpack.1 |        /Users/node_modules doesn't exist o
5:35:18 PM webpack.1 |  >  r is not a directory
5:35:18 PM webpack.1 |        /node_modules doesn't exist or is n
5:35:18 PM webpack.1 |  >  ot a directory
5:35:18 PM webpack.1 |   @ ./src/extensions/secret-manager/index.js 3:0-44 31:43-59
5:35:18 PM webpack.1 |   @ ./src/server/api/lib/config.js 4:0-63 69:17-26
5:35:18 PM webpack.1 |   @ ./src/lib/timezones.js 5:0-53 70:40-49
5:35:18 PM webpack.1 |   @ ./src/lib/index.js 3:0-235 3:0-235 3:0-235 3:0-235 3:0-23
5:35:18 PM webpack.1 |  >  5 3:0-235 3:0-235 3:0-235 3:0-235 3:0-235
5:35:18 PM webpack.1 |   @ ./src/client/error-catcher.js 1:0-29 4:4-13 7:2-11
5:35:18 PM webpack.1 |   @ ./src/client/index.jsx 5:0-43 11:2-14 14:2-14
5:35:18 PM webpack.1 |  webpack compiled with 1 error
```